extensions/industry/healthcare/fhir/datatypes/virtualservicedetail.schema.json

```diff
@@ -11,54 +11,52 @@
    "type":"object",
    "description":"Virtual Service Contact Details.(This datatype is created as per HL7 FHIR Release 5 specifications)",
    "definitions":{
-      "VirtualServiceDetail":{
```

Comment:
`VirtualServiceDetail` node seems to be deleted; we need it for the datatype to work correctly